### DISCUSSION TITLE
Simply saying thank you!!

### DISCUSSION THREAD

**Discussion by Dvalin21** [ZoeyVid/NPMplus#1039] [Q&A] [answered]
I will be watching his project from here on out and making suggestions when able. I just wanted to take the time to encourage you and say thank you as this was something LONG over due as far as the overhaul of npm. I've seen the php-rewrites and I'm excited about the changes. I'm not a coder and money is tight, but if I can keep you encouraged to continue with an encouraging word, absolutely!!!  Ill give when able.

**Comment by Zoey2936** [comment, marked as answer]
Thanks! php is currently a bit stuck, but I try to update vanilla and if possible also add new options

**Reply by Dvalin21** [reply]
You have a lot on your hands, but keep up the awesome work!
